hack/make.ps1

```diff
@@ -307,6 +309,44 @@ Function Validate-GoFormat($headCommit, $upstreamCommit) {
     }
 }
 
+Function Build-GoTestSum() {
```

Comment:
Should we build this as part of the Dockerfile, and then copy the binary out?

Reply:
@thaJeztah - Would it be something like so:

1. Introduce a new Dockerfile for building gotestsum
2. From the container that we currently execute unit tests in, use that Dockerfile to build gotestsum in another container 
3. Copy the built gotestsum binary back to the unit tests container.